docs/home/faq/terminology.mdx

```diff
@@ -23,6 +23,9 @@ slug: /faq/terminology
 - **Developers** - Individuals or organizations that build applications and services using the
   Helium Network infrastructure.
 
+- **DePIN** - DePIN, or Decentralized Physical Infrastructure Networks, projects use decentralized technology to power   
+  real-world uses, such as street maps, rendering power, cab-hailing services, and more. 
```

Comment:
The definition should probably include wireless networks since we're in the Helium docs.

Reply:
I have included wireless networks.